### DISCUSSION TITLE
ChatGPT reveals pro features

### DISCUSSION THREAD

**Discussion by levirtevs** [xyflow/xyflow#2951] [Ideas]
Hi,

I recently tried out chatGPT and I wanted to try out react flow pro features and it was able to provide me code.

I was concerned and wanted to drop a note.

-Vetri

**Comment by chrtze** [comment]
Hey @levirtevs, thanks for mentioning this. I'll try it myself.

**Comment by chrtze** [comment]
Do you have some example prompts that worked for you?

**Comment by ghost** [comment]
I tried this before (a month back ), and tried this after(today ) as well. but only time it worked, I documented few texts here. https://github.com/imvetri/ui-editor/issues/579
